pkg/kubeconfig/kubeconfig.go

```diff
@@ -72,6 +75,36 @@ func newConfigLoader(existing *cluster.Cluster) (*configLoader, error) {
 		sshconfig: sshConfig,
 	}, nil
 }
+
+func getRemoteFileBytes(address, filePath string, sshCfg *ssh.ClientConfig) ([]byte, error) {
+	conn, err := ssh.Dial("tcp", address, sshCfg)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Close()
+	c, err := sftp.NewClient(conn)
+	if err != nil {
+		return nil, err
+	}
+	defer c.Close()
+	r, err := c.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+	defer r.Close()
+	return ioutil.ReadAll(r)
+}
+
+func getRemoteKubeconfigPath(user string) string {
```

Comment:
This function could be simplified little bit, such as:
```
if user == "root" {
    return "/root/.kube/config"
}
return filepath.Join("/home", user, ".kube/config")
```
There is no need for another variable and more complex if/else logic.

Reply:
yes, thanks for the simplification. This looks so much better now. :)